proxy/src/schema/registry.rs

```diff
@@ -118,11 +130,25 @@ impl Registry {
             .await?
             .result?;
 
+        let register_metadata_vec = if let Some(pid_string) = maybe_pid {
+            let pid_cbor = ProjectCbor {
+                id: pid_string,
+                version: 1,
+            };
+            to_vec(&pid_cbor).expect("unable to serialize project metadata")
```

Comment:
We should actively work on removing panics in our modules. It shouldn't block any further progress on this PR, let's address it soon, maybe in the context of #137 - so maybe leave a `TODO` here.

Reply:
:+1: